### ISSUE TITLE
Issues with Ubuntu 15.10 and msp430g2 launchpad

### ISSUE THREAD

**Issue by cjdg** [energia/Energia#823]
I have issues working with the serial ports in linux, nothing works when i put the micro to sent serial data

**Comment by rei-vilo** [comment]
Could you please be more specific?

* Linux distribution
* Energia release
* LaunchPad reference and version
* USB connection —direct or [using hub](https://github.com/energia/Energia/wiki/Using-hub)

**Comment by cjdg** [comment]
Ubuntu 15.10 64 bit
Kernel version 4.2.0
Energia 17
msp430g2 1.5
As far as i know is an old issue with the kernel, with cdc_acm, which seems to dont get enough pressure to fix this, i was using patches, but after the last upgrade stop working.

**Comment by spirilis** [comment]
I've wrestled with this before, the main thing I had to keep in mind is DO NOT send any data from MCU -> Linux PC until you have a terminal app (or Serial Monitor) window open.  If you do, unplug the LaunchPad and plug it back in, then hold the RESET button until you get the serial monitor or terminal window open, then release RESET.

Alternately, organize your sketches to all check the P1.3 pushbutton in setup() before proceeding to initialize the Serial et al.
